exec/kubernetes_exec_manager.go

```diff
@@ -222,9 +223,22 @@ func (*KubernetesExecManager) Resize(id int, cols uint, rows uint) error {
 	return nil
 }
 
-func getById(id int) *model.MachineExec {
-	defer machineExecs.mutex.Unlock()
+// getK8sAPI return k8s api object.
+func (manager *KubernetesExecManager) getK8sAPI(userToken string) (k8s8API *client.K8sAPI, err error) {
+	if client.UseUserToken {
+		logrus.Info("Create k8s api object with user token")
+		k8s8API, err = manager.k8sAPIProvider.Getk8sAPIWithUserToken(userToken)
+	} else {
+		logrus.Info("Create k8s api object without user token")
+		k8s8API, err = manager.k8sAPIProvider.Getk8sAPI()
+	}
+	return
```

Comment:
I would avoid using return... I find it error producing and someday nil could be returned occasionally